features/FxA-I-11/README.md

```diff
@@ -5,50 +5,31 @@ Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeele
 
 Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
 
-## Outcomes
+## Problem Summary
 
-The success of this feature is being able to answer the question:
+There are a lot of errors related to password entry.
```

Comment:
I wonder if we could add some additional information about the goal of the system and how the password errors negatively affect the user experience.

`Currently, there are a high incident of errors related to password entry. With the push for stronger and more secure passwords these errors will continue to be prevalent.` or something along those lines?